ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -103,15 +103,36 @@ If.dmic {
 		Type String
 		Empty "${var:DeviceDmic}"
 	}
-	False.If.Dmic0 {
-		Condition {
-			Type ControlExists
-			Control "name='Dmic0 Capture Volume'"
+	False {
```

Comment:
@perexg I tried to make these additions to If.dmic subtree copy in HiFi-sof.conf but I could not get it working. Do you have ideas why? Basically this same add but without duplicate Dmic0 Capture Volume part.

I'd prefer to have this in HiFi-sof.conf to have access to Blob name variables. I checked with echo that they are not here in sof-hda-dsp.conf. Then I could add as FixedBootSequence load (in every device boot) of blob for microphone DRC and TDFB. It would allow to override the generic blobs embedded to topology with DMI ID match based tuned blobs for the device.

Reply:
BootSequences are allowed only in the top level configuration file. But you can move variable declarations from the HiFi file to the top-level file. The variables are shared for all configuration files.

Reply:
Thinking more - HiFi-sof.conf should be splitted to two parts - e.g. `dsp.conf` and `HiFi-sof.conf` where dsp.conf will declare the variables and this file should be included from `sof-hda-dsp.conf`.

Reply:
Thanks! I'll try that.

Reply:
@perexg Here's now my try to follow the idea. What do you think?